openlibrary/templates/search/sort_options.html

```diff
@@ -0,0 +1,28 @@
+$def with (exclude, default)
+
+$ sort = 'sort'
+
+<span class="tools"><img src="/images/icons/icon_sort.png" alt="$_('Sorting by')" width="9" height="11" style="margin-right:10px;"/>
+    $code:
+      sort_options = [
+        { 'sort': None, 'name': _("Relevance"), 'ga_key': 'Relevance' },
+        { 'sort': 'editions', 'name': _("Most Editions"), 'ga_key': 'Editions' },
+        { 'sort': 'old', 'name': _("First Published"), 'ga_key': 'Old' },
+        { 'sort': 'new', 'name': _("Most Recent"), 'ga_key': 'New' },
+        { 'sort': 'random', 'name': _("Random"), 'ga_key': 'Random', 'selected': sort and sort.startswith('random') },
+      ]
+    $for sort_option in sort_options:
+      $ is_selected = sort_option.get('selected') or sort_option['sort'] == sort
+      $if is_selected:
+          <strong class="lightgreen">$sort_option['name']</strong>
+      $else:
+          <a href="$changequery(sort=sort_option['sort'])"
```

Comment:
and here, because we want `sort=None` if it's the default, we'll have to do something like
```suggestion
          <a href="$changequery(sort=cond(sort_option['sort'] == default_sort, None, sort_option['sort']))"
```